interface/fax/fax_dispatch.php

```diff
@@ -614,203 +594,216 @@ function allCheckboxes(issel) {
+                <input type='radio' name='form_cb_copy_type' value='2'
+                    onclick='return divclick(this,"div_copy_sn");' />
+                <label class="font-weight-bold"><?php echo xlt('Scanned Encounter Note'); ?></label>
+            <?php } ?>
+            <!-- div_copy_doc Section -->
+            <div id='div_copy_doc' class='bg-secondary border rounded p-2'>
+                <!-- Category Section -->
+                <div class="form-row">
+                    <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Category'); ?></label>
+                    <div class="col-10">
+                        <select name='form_category' class='form-control'>
+                            <?php
+                            foreach ($categories as $catkey => $catname) {
+                                echo "         <option value='" . attr($catkey) . "'";
+                                echo ">" . text($catname) . "</option>";
+                            }
+                            ?>
+                        </select>
+                    </div>
+                </div>
+                <!-- Filename Section -->
+                <div class="form-row">
+                    <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Filename'); ?></label>
+                    <div class="col-10">
+                        <input type='text' size='10' name='form_filename' class='form-control'
+                            value='<?php echo attr($filebase) . ".pdf" ?>'
+                            data-toggle='tooltip' data-placement='top'
+                            title='Name for this document in the patient chart' />
+                    </div>
+                </div>
+                <!-- Document Date Section -->
+                <div class="form-row">
+                    <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Document Date'); ?></label>
+                    <div class="col-10">
+                        <input type='text' class='datepicker form-control' size='10' name='form_docdate' id='form_docdate'
+                            value='<?php echo date('Y-m-d'); ?>'
+                            data-toggle='tooltip' data-placement='top'
+                            title='<?php echo xla('yyyy-mm-dd date associated with this document'); ?>' />
+                    </div>
+                </div>
+            </div>
+            <!-- div_copy_sn Section -->
+            <div id='div_copy_sn' class='bg-secondary border rounded p-2' style='display:none;margin-top:0.5em;'>
+                <!-- Visit Date Section -->
+                <div class="form-row">
+                    <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Visit Date'); ?></label>
+                    <div class="col-10">
+                        <select name='form_copy_sn_visit' class='form-control'>
+                        </select>
+                    </div>
+                </div>
```

Comment:
Hi @tywrenn 

After I changed to the `form-row` it looks like need some margin e.g. `mt-2`. What do you think?

![image](https://user-images.githubusercontent.com/22230889/82844437-02e55380-9f13-11ea-871b-bf76a7150a4b.png)

Reply:
@stu01509 yeah it'll need some margins with it

Reply:
Update the commit, Looks like good :)

![image](https://user-images.githubusercontent.com/22230889/82847325-14812800-9f20-11ea-8160-efbe0145f93e.png)